packages/ui/src/theme/tokens/components/copy.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  fontSize: { value: '{font.sizes.xs.value}' },
+  gap: { value: '{space.relative.medium.value}' },
+
+  svg: {
+    path: { fill: { value: '{space.relative.medium.value}' } },
+  },
+  toolTip: {
+    bottom: { value: '{space.large}' },
+    color: { value: '{colors.teal.100}' },
+    fontSize: { value: '{font.sizes.xxs}' },
```

Comment:
This is missing `.value` at the end of the reference.

Reply:
same here. I didn't use `.value` on purpose because it saves several stokes with out changing anything and I'm lazy 👻